.travis.yml

```diff
@@ -57,28 +59,55 @@ matrix:
             rm -rf "$TAPS/caskroom/homebrew-cask";
           fi;
           if [ ! -f $HOME/.osx_cache/cached ]; then
-            brew tap homebrew/homebrew-cask;
+            echo "==> [Not found cache] brew tap homebrew/homebrew-cask";
+            echo "[NOTE]";
+            echo "If brew is executed without HOMEBREW_NO_AUTO_UPDATE=1,";
+            echo "python3 cannot be installed, so this is added as a temporary workaround.";
+            echo "If it is xcode 9.4 or higher, clear this patch.";
+            HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/homebrew-cask;
           else
+            echo "==> [Found cache] HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/homebrew-cask";
             HOMEBREW_NO_AUTO_UPDATE=1 brew tap homebrew/homebrew-cask;
           fi
         - HOMEBREW_NO_AUTO_UPDATE=1 brew cask install osxfuse
-        - S3FS_BREW_PACKAGES='awscli cppcheck truncate';
+        - S3FS_BREW_PACKAGES='cppcheck python3';
```

Comment:
Needed?

Reply:
This is because I want to install the python3 code with awscli install if possible.
I think awscli will not support python2 either soon.